app/javascript/forms/mappers/validatorMapper.jsx

```diff
@@ -0,0 +1,13 @@
+import { parseCondition } from '@data-driven-forms/react-form-renderer';
+import validatorMapper from '@data-driven-forms/react-form-renderer/dist/cjs/validator-mapper';
```

Comment:
I know, this is ugly :vomiting_face: and it should be probably exported properly cc @Hyperkid123

Reply:
This is not ugly and that's how it actually recommended importing way of dependencies if you want to tree-shake it properly :smile:  :dagger: .

I guess it is missing in the index.js root file. We can fix that easily.

Reply:
@Hyperkid123 my problem is the `cjs` in the path

Reply:
@skateman you can use `/esm/` if you like these letters better :smile: . I will add them to the index so you can use the shorthand as well. It should be there anyway if there is a direct import path.